drivers/mm/mm_drv_intel_adsp_mtl_tlb.c

```diff
@@ -693,4 +698,162 @@ static int sys_mm_drv_mm_init(const struct device *dev)
 	return 0;
 }
 
-SYS_INIT(sys_mm_drv_mm_init, POST_KERNEL, 0);
+#ifdef CONFIG_PM_DEVICE
+static void sys_mm_save_context(void *storage_buffer)
```

Comment:
Please do not use the `sys_mm_` namespace for these newly added functions as these are not official APIs.

Reply:
@dcpleung please give me some suggestions on function names if the new ones turn out to be bad.

Reply:
New names look good.